### PR TITLE
Pt/qa

### PR DESCRIPTION
**Pull request by paaaatrick**
## Trello Card(s)
https://trello.com/c/KOZrpgEI/35-basic-page-qa-items
https://trello.com/c/cQHEk9Py/36-callout-grid-qa
https://trello.com/c/a0oOaEDP/34-no-breadcrumb-on-basic-page
## What was updated?
- content no longer extends outside of background on callout grid.
- small amount of padding added to header
- chevrons in ie-11 no longer overlap text
- added author byline field to news
- added help text to advanced settings. 
- added breadcrumbs to basic pages
- revises header to use mobile nav on tablet and smaller large screens.
## Screenshots of updates (visual changes)

author byline to news
Screenshot 2019-12-09 17 10 32

help text to advanced settings paragraph
Screenshot 2019-12-09 17 15 38

adds breadcrumbs to basic pages
Screenshot 2019-12-10 09 24 02

revises header to use mobile nav on smaller large screen sizes. 
Screenshot 2019-12-10 10 54 06

### CODE REVIEW

File: config/field.field.paragraph.advanced_settings.field_mobile_gap.yml [hunk 1]
```diff
@@ -19,5 +19,5 @@ settings:
   min: null
   max: null
   prefix: ''
-  suffix: ''
+  suffix: px
```
Comment (theroadeldorado) [inline config/field.field.paragraph.advanced_settings.field_mobile_gap.yml]: remove this

File: config/field.field.paragraph.advanced_settings.field_desktop_gap.yml [hunk 2]
```diff
@@ -19,5 +19,5 @@ settings:
   min: null
   max: null
   prefix: ''
-  suffix: ''
+  suffix: px
```
Comment (theroadeldorado) [inline config/field.field.paragraph.advanced_settings.field_desktop_gap.yml]: remove this

File: config/core.entity_form_display.paragraph.callout.default.yml [hunk 3]
```diff
@@ -3,38 +3,17 @@ langcode: en
...
   module:
     - link
-    - paragraphs
     - text
 id: paragraph.callout.default
 targetEntityType: paragraph
 bundle: callout
 mode: default
 content:
-  field_advanced_settings:
```
Comment (theroadeldorado) [inline config/core.entity_form_display.paragraph.callout.default.yml]: You are removing this I don't think that you want too.

File: config/field.field.paragraph.advanced_settings.field_desktop_gap.yml (hunk 2 above)
Comment (paaaatrick) [reply config/field.field.paragraph.advanced_settings.field_desktop_gap.yml]: removed

File: config/core.entity_form_display.paragraph.callout.default.yml (hunk 3 above)
Comment (paaaatrick) [reply config/core.entity_form_display.paragraph.callout.default.yml]: added back

**Pull request merged**